types/knex.d.ts

```diff
@@ -829,6 +829,7 @@ declare namespace Knex {
     extension?: string;
     tableName?: string;
     disableTransactions?: boolean;
+    schemaName?: string;
```

Comment:
since you are changing it anyway, could you please also add

`sortDirsSeparately : optional boolean`
and update `directory` one to support both String and String[]?

Reply:
Absolutely !

There is other diff with Doc :
- `loadExtensions`, `migrationSource` are in doc, not in code
- `database` is in code, not in doc.